src/xcode/ENA/ENA/Source/Services/Risk/Calculation/__test__/Models/RiskCalculationResultTests.swift

```diff
@@ -0,0 +1,52 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+@testable import ENA
+
+class RiskCalculationResultTests: XCTestCase {
+
+	func testGIVEN_NewFormatRiskCalculationResultWithoutRiskLeevlPerDay_WHEN_ParseJson_THEN_RiskCalculationResultGetsCreated() {
+		// GIVEN
+		let newFormattedData = "{\"minimumDistinctEncountersWithHighRisk\":0,\"riskLevelPerDate\":[],\"numberOfDaysWithLowRisk\":0,\"calculationDate\":632684966.21907699,\"numberOfDaysWithHighRisk\":0,\"minimumDistinctEncountersWithLowRisk\":0,\"riskLevel\":1}"
+			.data(using: .utf8)
+
+		// WHEN
+		let riskCalculationResult = try? JSONDecoder().decode(RiskCalculationResult.self, from: XCTUnwrap(newFormattedData))
+
+		// THEN
+		XCTAssertNotNil(riskCalculationResult)
+		XCTAssertEqual(riskCalculationResult?.riskLevelPerDate.count, 0)
+	}
+
+	func testGIVEN_NewFormatRiskCalculationResultWithThreeRiskLeevlPerDay_WHEN_ParseJson_THEN_RiskCalculationResultGetsCreated() {
```

Comment:
I would prefer a proper documentation over the concatenated implementation code as function name.

Reply:
I added some comments